src/components/InputForm.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+    <div>
+        <p class="errorText">{{errorText}}</p>
+        <p class="padding">Номер телефон</p>
+        <label>
+            <input type="tel" v-mask="'+7(###) ###-##-##'" v-model="phoneNumber"
+                   placeholder="Введите Ваш номер телефона">
+        </label>
+        <p>Сумма в рублях</p>
+        <label>
+            <input type="tel" v-mask="'### рублей'" v-model="depositAmount" placeholder="Введите сумму в рублях">
+        </label>
+        <div class="row">
+            <button @click="backClick">На главную</button>
+            <div style="width: 32px"/>
+            <button @click="payClick">Оплатить</button>
+        </div>
+    </div>
+</template>
+
+<script>
+    import router from "@/router";
+
+    export default {
+        name: "InputForm",
+        props: ['operatorLabel', 'pictPath'],
+        methods: {
+            backClick() {
+                router.back()
+            },
+            payClick() {
+                if (this.phoneNumber.length < 17) {
+                    this.errorText = "Укажите правильный номер телефона"
+                } else if (this.depositAmount.length === 0) {
```

Comment:
if-else statements might waste readability, consider avoid them 

the first results in google for query "avoid if-else"
https://hackernoon.com/stop-using-the-else-keyword-in-your-code-9m7x3y97
https://www.digitalocean.com/community/posts/5-tips-to-write-better-conditionals-in-javascript